src/json.cpp

```diff
@@ -12,6 +12,8 @@
 #include <vector>
 #include <bitset>
 
+#include "item.h"
```

Comment:
This disables the chkjson binary. It only contains the chkjson.cpp and json.cpp, but does not include the other source files (and therefor the reference to `item` will be missing).